internal/renew_root.go

```diff
@@ -45,6 +50,15 @@ func (s fetchRootStep) Name() string {
 
 func (s fetchRootStep) Execute(h *rootRenewalContext) error {
 	caFile := h.app.sota.GetOrDie("import.tls_cacert_path")
+	caCertBuf, err := os.ReadFile(caFile)
+	if err != nil {
+		log.Fatal("Failed to read root CA file", err)
+	}
+	caCerts, err := loadCertsFromPem(caCertBuf)
+	if err != nil {
+		log.Fatal("Failed to parse root CA file", err)
```

Comment:
this should return an error and not log.Fatal, right?

Reply:
A root renewal script is not able to execute if it cannot parse CA certs.
So, it's a fatal error.